src/main/msp/msp.c

```diff
@@ -1360,6 +1360,15 @@ static bool mspProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 0);
 #endif
         break;
+
+    case MSP_RXRANGE_CONFIG:
+        for (int i = 0; i < NON_AUX_CHANNEL_COUNT; i++) {
```

Comment:
Add a leading byte that indicates the following value pair count so the consumer can process correctly. If `NON_AUX_CHANNEL_COUNT` was to change in a later version then any clients would break.